app/spdb/build.sbt

```diff
@@ -250,12 +250,12 @@ def dnavarro(version: Version) = AppConfig(
     "edu.gemini.auxfile.fits.dest"               -> "/gemsoft/var/data/ictd/test/GS@SEMESTER@/@PROG_ID@",
     "edu.gemini.auxfile.fits.host"               -> "gsconfig.gemini.edu",
     "edu.gemini.auxfile.other.dest"              -> "/gemsoft/var/data/finder/GSqueue/Finders-Test/@SEMESTER@/@PROG_ID@",
-    "edu.gemini.auxfile.root"                    -> "/Users/jnavarro/.auxfile",
+    "edu.gemini.auxfile.root"                    -> "/Users/dnavarro/.auxfile",
```

Comment:
@cquiroz I have used the spdb in the past when working on a task for the OT, and I guess I may need to use it at some point in the future. I can add my credentials myself, as you created them for me a while ago. I imagine that it would be better not to do it in a commit in this branch. What would be the right one?

@jdnavarro, sorry for the mess, I'll fix the build file and send a new pull request.